Code/Waveforms.cpp

```diff
@@ -663,6 +663,14 @@ std::string GWFrames::Waveform::DescriptorString() const {
       Descriptor = Descriptor + DataTypeString(); // hdot is independent of M
     else if(DataType()==Psi4)
       Descriptor = Descriptor + "M" + DataTypeString();
```

Comment:
Shouldn't you add corresponding cases to the `if(RIsScaledOut())` section, using `r2` for `Psi3`, `r3` for `Psi2`, etc?